titles/discover/master.adoc

```diff
@@ -0,0 +1,16 @@
+include::artifacts/attributes.adoc[]
+:context: title-discover
+:doctype: book
+:imagesdir: images
+:title: About
```

Comment:
Consider using the same words for the directory name (now "discover") and the title (now "About").